spec/lib/rbac/filterer_spec.rb

```diff
@@ -2459,8 +2459,8 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
     end
 
     it "support aaarm object" do
-      expect(LiveMetric).to receive(:find).with(:all, :include => {:a => {}}).and_return([:good])
-      expect(described_class.filtered(LiveMetric, :include_for_find => {:a => {}}).to_a).to match_array([:good])
+      expect(VimPerformanceTrend).to receive(:find).with(:all, :include => {:a => {}}).and_return([:good])
+      expect(described_class.filtered(VimPerformanceTrend, :include_for_find => {:a => {}}).to_a).to match_array([:good])
```

Comment:
This was a valid test, and just happened to use `LiveMetric` for it so I just picked another `ActsAsArModel` class